src/components/channel/ChannelBadge.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+
+import { SubscribeButton } from './SubscribeButton'
+
+export const ChannelBadge = ({
+  children,
+  className = '',
+  channel,
+  ...rest
+}) => (
+  <div className={`channel-badge ${className}`} {...rest}>
+    <div className='thumbnail-wrapper'>
+      <img className='thumbnail' src='http://placehold.it/60x60' />
```

Comment:
The image URL will actually be part of the Channel model. Could you move the placeholder into the storybook data?